knowledge-base/anchore/sbom-action/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: "Anchore SBOM Action"
+github-token:
+  action-input:
+    input: github-token
+    is-default: true
+  permissions:
+    actions: write
+    actions-reason: to upload & delete workflow artifacts & assets  # CHECKOUT: https://github.com/anchore/sbom-action/blob/3626d7d7b13e87ee6c6f9ded3940dea05a3967bc/src/github/SyftGithubAction.ts#L262
```

Comment:
@h0x0er I did not see it deleting workflow artifact. You will need to split this into two
1. actions: write for uploading workflow artifact
2. contents: write for uploading release artifact

Reply:
@varunsh-coder I have made the required changes.
As per here [here](https://github.com/anchore/sbom-action/blob/3626d7d7b13e87ee6c6f9ded3940dea05a3967bc/src/github/SyftGithubAction.ts#L459), the action is deleting release assets